mapview_toolchain.Rmd

```diff
@@ -8,15 +8,15 @@ output:
+An integral part of any spatial data analysis workflow is visual inspection of intermediate analysis results. Virtually every open or closed source geospatial software provides an interactive data visualisation module where the visualisation is updated throughout each of the steps in the analysis. These visual modules usually provide easy access to the data through layered plots that can be changed and explored with one or two mouse clicks. This makes visual exploration of complex spatial data easy and quick. Such quick visual inspection capabilities are currently lacking in R. The package [mapview](https://cran.r-project.org/web/packages/mapview/index.html) was created to fill this gap. It is in large parts built on top of the `leaflet` package and enables interactive visualisation of multiple layers with one or only a few lines of code, e.g. `mapview(x, burst = TRUE) + y` will visualise all attributes of spatial object `x` as individual layers and additionally add a layer of the locations of object `y`. Furthermore, all features of `x` and `y` are fully queryable via popups. As such, it provides an interactive version of `plot()` or `spplot()` for spatial data. Spatial data can quickly grow to large proportions of many tens of thousands of features, so `mapview` also includes some special functionality to cope with such amounts.  `Leaflet` based maps can become rather unresponsive when data sets are "big".
 
 So far, we have outlined why visualisation is an important part of geospatial data analysis and have shown several possiblities how it can be achieved. Another aspect of geospatial analysis, which is implemented in all open or closed source geospatial software suites is interactive data manipulation. This is what this application is intended to address.
```

Comment:
Should this be 3, or is this just repetition of 2 above from copy/paste?  If 3, should two-fold above be changed to three-fold?